src/Sulu/Bundle/AdminBundle/Resources/js/components/FolderList/README.md

```diff
@@ -0,0 +1,62 @@
+The FolderList component is used display a list of folders. The FolderList accepts Folder componenta as children.
+The Folder component is just an interactive element for presentation purposes. Every Folder component has its
+own onClick-handler. The FolderList component offers a more convenient way to work with multiple folders by offering 
+a single onClick-handler.
```

Comment:
Please use backticks if you are referencing code, like for `onClick`, `Folder` or `FolderList`